rsync_to_rdisc.py

```diff
@@ -234,11 +241,90 @@ def rsync_server_remote(settings, hpc_server, client, to_be_transferred):
             log_file.write("\n#########\nDate: {date}\nRun_folder: {run}\n".format(date=date, run=run))
 
         if continue_rsync:
-            rsync_and_check(action, run, to_be_transferred[run], temperror, settings.wkdir, folder_dic, log)
+            rsync_result = rsync_and_check(action, run, to_be_transferred[run], temperror, settings.wkdir, folder_dic, log)
+            if rsync_result == "ok":
+                upload_result_gatk = None
+                upload_result_exomedepth = None
+                if folder['upload_gatk_vcf']:
+                    upload_result_gatk = upload_gatk_vcf(
+                        run=run,
+                        run_folder="{output}/{run}".format(output=folder["output"], run=run)
+                    )
+                if folder['upload_exomedepth_vcf']:
+                    upload_result_exomedepth = upload_exomedepth_vcf(
+                        run=run,
+                        run_folder="{output}/{run}".format(output=folder["output"], run=run)
+                    )
+                make_mail(
+                    filename="{}{}".format(folder["input"], run),
+                    state="ok",
```

Comment:
state = rsync_result?  Will prevent 2 changes if we ever decide to change "ok" ;)